src/components/ButtonTryAgain/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import './button-try-again.css';
+const ButtonTryAgain = (props) => {
+    return (
+        <button className="button-try-again" onClick={ props.handleClick }>try again</button>
+    )
+}
+
+export default ButtonTryAgain;
```

Comment:
You can write `export const ButtonTryAgain = (props) => {}` instead of `export default ButtonTryAgain; `